src/muya/lib/utils/dirtyToc.js

```diff
@@ -30,7 +30,8 @@ export const slugify = str => {
   })
 
   return str
-    .replace(/[^a-z0-9 -]/g, '') // remove invalid chars
+    // add support Chinese \u4e00-\u9fa5
+    .replace(/[^a-z0-9\u4e00-\u9fa5 -]/g, '') // remove invalid chars
```

Comment:
The slug function must return an ASCII string which is valid as HTML class name. Currently we're using [diacritics-map](https://github.com/jonschlinkert/diacritics-map) to map non-ASCII characters but it doesn't support Russian, Chinese etc characters (see line 27). We could use [this map](https://github.com/cocur/slugify/blob/master/src/RuleProvider/DefaultRuleProvider.php) or maybe replace our slug function with [slugify](https://github.com/simov/slugify)?

Reply:
Can you make a PR of this? Does slugify support Chinese?

Reply:
> Can you make a PR of this?

Yes, I can.

> Does slugify support Chinese?

Unfortunately not but it seemed so.